### PR TITLE
Moved all of the logging logic inside engine.logs

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1430809.
The tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/1099

### CODE REVIEW

File: openquake/engine/tests/utils/helpers.py [hunk 1]
```diff
@@ -533,66 +533,3 @@ def create_ses_ruptures(job, ses_collection, num):
             rupture=pr, ses_id=ses_ordinal, tag=tag, seed=seed + i)
         sesruptures.append(sr)
     return sesruptures
-
-
```
Comment (micheles) [inline openquake/engine/tests/utils/helpers.py]: The class below was duplicated

File: openquake/engine/logs.py [hunk 2]
```diff
@@ -153,17 +164,28 @@ def handle(job, log_level='info', log_file=None):
...
     try:
         yield
     finally:
-        logging.root.removeHandler(handler)
-        logging.root.removeHandler(db_handler)
+        for handler in handlers:
+            logging.root.removeHandler(handler)
+        # sanity check to make sure that the logging on file is working
+        if log_file:
+            assert os.path.getsize(log_file) > 0
```
Comment (ptormene) [inline openquake/engine/logs.py]: Please use a warning instead of an assert

**Comment by ptormene** [comment]
Apart from the comment, LGTM

File: openquake/engine/logs.py (hunk 2 above)
Comment (micheles) [inline openquake/engine/logs.py]: Done

**Pull request merged**